releases/cloud/point-2024-02-22-release-note.md

```diff
@@ -0,0 +1,47 @@
+---
+description: >-
+  This article lists improvements and bug fixes in the Syskit Point Cloud version 2024.1.43.10
+---
+
+# February 22, 2024
+
+[Start a free trial](https://www.syskit.com/products/point/free-trial/) and [tell us what you think](https://www.syskit.com/company/contact-us/).
+
+
+## About Syskit Point Cloud
+
+* **New to Syskit Point Cloud?** Explore Syskit Point Cloud with a 21-day [free trial](https://www.syskit.com/products/point/free-trial/) for an easy and effective way to manage and secure your environment.
+* **Already using Syskit Point Cloud?** Syskit Point Cloud is automatically upgraded to the latest version when available. The automatic update occurs outside working hours to ensure minimal interference with your day-to-day business.
+
+
+## Improvements & Bug Fixes
+
+* **The Change Storage Limit action** was added to the Storage Metrics report.
+
+* **The Remove Access action was enabled** for users in the Access review task on the Sharing step - Users View.   
+  * Multiselection of users is also enabled.
+
+* **Every file** now has the sensitivity and retention labels applied when they are available. 
+
+* **Improved the performance** of the Site Storage Flat Report and Sure Storage Structure Report. 
+
+* **Improved the performance** of the 'Tasks Overview' page when showing a large number of tasks.  
+
+* **Fixed the issue** that caused the Permissions Summary report on the Site Details screen to time out in some environments.
+
+* **Fixed an error** that caused a Duplicate ID crash when sending Lifecycle Management reminders.
```

Comment:
ovo malo bolje objasniti. 
- nije jasno što je Duplicate ID crash
- navesti s kojeg screena se šalje reminder pa će biti lakše za shvatiti